pkg/proxy/userspace/proxier.go

```diff
@@ -510,20 +571,70 @@ func (proxier *Proxier) unmergeService(service *v1.Service, existingPorts sets.S
 	}
 }
 
+func (proxier *Proxier) processServiceEvent(event serviceEventType, oldService, service *v1.Service) {
+	e := serviceEvent{
+		event:      event,
+		service:    service.DeepCopy(),
+		oldService: oldService.DeepCopy(),
+	}
+
+	proxier.mu.Lock()
+	proxier.serviceEvents = append(proxier.serviceEvents, e)
+	// Sync immediately to reduce memory usage if we get too many events
+	syncNow := len(proxier.serviceEvents) > maxServiceEvents
```

Comment:
This seems to defeat the optimization of bounded-frequency?  E.g. I could still spam you with meaningless Service changes and pound on iptables..